### DISCUSSION TITLE
missing initial state

### DISCUSSION THREAD

**Discussion by Mrsanchaari** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#718] [Q&A]
I need help with this error as I can't generate a toke
![Screenshot 2022-05-18 112951](https://user-images.githubusercontent.com/105771653/168968948-4efe9efd-d917-4b9b-af09-48d3be565000.png)
n

**Comment by sumeshir26** [comment]
Copy the code from the address bar
